src/components/UrqlSubscription.re

```diff
@@ -1,11 +1,10 @@
-[@bs.module "urql"]
-external subscriptionComponent: ReasonReact.reactClass = "Subscription";
-
+[@bs.deriving abstract]
```

Comment:
There are some follow up changes to make here, but I'm going to make a separate issue to track making the components type safe in the same manner as the hooks.